src/features/deployEcosystem/application/waitUntilTxIsConfirmed.ts

```diff
@@ -35,7 +36,7 @@ async function isTxConfirmed(
 
   const confirmations = currentBlock - receipt.blockNumber;
 
-  const success = confirmations >= 3;
+  const success = confirmations >= config.confirmations;
```

Comment:
Pretty sure that `1 >= undefined === false`?

I'd suggest that we set the default in `appSettings` for this to 3 so that it's never `undefined`